src/layers/Layer.js

```diff
@@ -343,7 +343,8 @@ class Layer extends Evented {
             const { properties } = feature
             const content = (hoverLabel || label).replace(
                 /\{ *([\w_-]+) *\}/g,
-                (str, key) => properties[key] || ''
+                (str, key) =>
+                    properties[key] || (key === 'value' ? 'No data' : '')
```

Comment:
Is the translated string for "No data" being passed from maps-app? So that this "No data" on line 347 is just a backup for edge cases

Reply:
That's a good point. As far as I know, this implementation does not support translation. I'll look into supporting it.

Reply:
Addressed with:
- in maps-gl: https://github.com/dhis2/maps-gl/pull/573/commits/c20840261940d8cd5047d4855725edc48d1513f6 + https://github.com/dhis2/maps-gl/pull/573/commits/15b5e3b58c667a4e70ad55ffe687432be9d23080
- in maps-app: https://github.com/dhis2/maps-app/pull/3204/commits/fa21fbcf92ec4614b6dd1c5ded4c0d03256e95ed